schema/Makefile

```diff
@@ -71,7 +71,7 @@ deploy:
 .PHONY: prove_style
 prove_style:
 	# Run style-related test suite on all objects in db using pg_prove
-	@@$(PG_PROVE) --failures -d $(TEST_DB) test/style/*_test.sql
+	@@$(PG_PROVE) --failures -d $(TEST_DB) test/style/*_test.sql --set schemas_to_test=ggircs_portal,ggircs_portal_private --set dummy_schema=test_fixture_schema --set audit_schema=audit
```

Comment:
`dummy` is on the [list of terms to avoid](https://developers.google.com/style/word-list?hl=en), please find an alternative name (`placeholder_schema`,`temp_schema`,...). Should this one even be a variable?

Reply:
good point, dummy_schema and audit_schema do not need to be variables